latex_cite_completions.py

```diff
@@ -783,10 +664,28 @@ def on_done(i):
             view.sel().subtract(view.sel()[0])
             view.sel().add(sublime.Region(caret, caret))
         else:
+            # get preferences for formating of quick panel
+            cite_panel_format = get_setting('cite_panel_format',
+                ["{title} ({keyword})", "{author}"])
             # show quick
-            formatter = Formatter()
-            view.window().show_quick_panel([[formatter.vformat(s, (), completion) for s in cite_panel_format] \
-                                        for completion in completions], on_done)
+            def formatted_entry(entry):
+                try:
+                    return entry["<panel_formatted>"]
+                except:
+                    return [formatted_entry(s, entry) for s in cite_panel_format]
```

Comment:
Isn't this an infinite loop? It should probably be `bibformat.format_entry(s, entry)`